zappa/cli.py

```diff
@@ -422,7 +422,7 @@ def schedule(self):
                 events = []
             keep_warm_rate = self.zappa_settings[self.api_stage].get('keep_warm_expression', "rate(5 minutes)")
             events.append({'name': 'zappa-keep-warm',
-                           'function': self.lambda_handler,
+                           'function': 'handler.heater',
```

Comment:
Why are you doing this?

The goal of this is to call the web function handler so that the function is kept warm.